aerogear/unifiedpush-operator
# Updating Enmasse to work with AMQ Online 1.4

## Motivation
AMQ Online has been updated to 1.4 which fixes a lot of issues we've had.  This PR brings those updates to unifiedpush-operator.
## What
 * enmasse was updated to the latest version in master 
 * type related code breaks were fixed, but they didn't change logic.

## Why
There were many issues with AMQ Online.  The ones that affected us were
 * infinite logs with bad watches (this is fixed in 1.4)
 * addresses hanging on creation (this should have been fixed in 1.4 as well)



## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Install this operator on a cluster with AMQ Online 1.4
2. Create a ups resource with useMessageBroke : true
3. Ensure that the ups, messaging user, addresspace, and addresses all are created and running
4. Ensure that UPS works as normal
## Checklist:

- [ x ] Code has been tested locally by PR requester

## Progress


## Additional Notes

I can set up a cluster if necessary; the code was run locally.
